### PR TITLE
feature: audience/scope for access token

### PR DESCRIPTION
**Pull request by minh-rakuten**
# Description
Merge access token into master branch.

# Checklist
- [x] I have read the [contributing guidelines](../.github/CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](../.github/CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [x] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [ ] I ran `./gradlew check` without errors => I got error in local while running unit tests.

### CODE REVIEW

File: miniapp/USERGUIDE.md [hunk 1]
```diff
@@ -367,7 +368,7 @@ The following user data types are supported. If your App does not support a cert
...
 - Profile photo: URL pointing to a photo. This can also be a Base64 data string. See [UserInfoBridgeDispatcher.getProfilePhoto](api/com.rakuten.tech.mobile.miniapp.js.userinfo/-user-info-bridge-dispatcher/get-profile-photo.html)
-- Access Token (does not currenlty have a custom permission type): OAuth 1.0 token including token data and expiration date. Your App will be provided with the ID of the mini app which is requesting the Access Token, so you should verify that this mini app is allowed to use the access token. See See [UserInfoBridgeDispatcher.getAccessToken](api/com.rakuten.tech.mobile.miniapp.js.userinfo/-user-info-bridge-dispatcher/get-access-token.html)
+- Access Token: OAuth 1.0 token including token data and expiration date. Your App will be provided with the ID of the mini app and [AccessTokenPermission]((api/com.rakuten.tech.mobile.miniapp.permission/-access-token-permission)) which is requesting the Access Token, so you should verify that this mini app is allowed to use the access token. See [UserInfoBridgeDispatcher.getAccessToken](api/com.rakuten.tech.mobile.miniapp.js.userinfo/-user-info-bridge-dispatcher/get-access-token.html)
```
Comment (corycaywood) [inline miniapp/USERGUIDE.md:371]: Looks like this one needs to be updated to the correct link: `api/com.rakuten.tech.mobile.miniapp.permission/-access-token-scope`

**Review by corycaywood — APPROVED** [review]
LGTM 👍

File: miniapp/USERGUIDE.md (hunk 1 above)
Comment (minh-rakuten) [reply miniapp/USERGUIDE.md:371]: Good catch!

**Review by khairul-alam-licon — APPROVED** [review]
Looks good 👍

File: miniapp/USERGUIDE.md (hunk 1 above)
Comment (Climbatize) [reply miniapp/USERGUIDE.md]: @corycaywood you are a machine

**Pull request merged**